src/mod/endpoints/mod_sofia/sofia.c

```diff
@@ -5110,13 +5110,13 @@ switch_status_t config_sofia(sofia_config_t reload, char *profile_name)
 						}
 
 						if (strchr(ip, ':')) {
-							if (profile->rtpip_index < MAX_RTPIP) {
+							if (profile->rtpip6_index < MAX_RTPIP) {
```

Comment:
Should be rtpip_index6, not rtpip6_index

Reply:
uh-oh you're right :D 
fixed.